GliaWidgetsTests/ViewModel/Chat/ChatMessagaEntryViewTests.swift

```diff
@@ -0,0 +1,30 @@
+@testable import GliaWidgets
+import XCTest
+
+class ChatMessageEntryViewTests: XCTestCase {
+    let messagageEntryView = ChatMessageEntryView.mock()
```

Comment:
Since `messagageEntryView` is shared across tests, you need to erase its state in `setUp` method